runtime/catalog-karapace/src/test/java/io/aklivity/zilla/runtime/catalog/karapace/internal/KarapaceIT.java

```diff
@@ -72,7 +73,7 @@ public void shouldResolveSchemaViaSchemaId() throws Exception
             "{\"name\":\"status\",\"type\":\"string\"}]," +
             "\"name\":\"Event\",\"namespace\":\"io.aklivity.example\",\"type\":\"record\"}";
 
-        KarapaceCatalogHandler catalog = new KarapaceCatalogHandler(config, context, 0L);
+        KarapaceCatalogHandler catalog = new KarapaceCatalogHandler(config, context, 0L, new ConcurrentHashMap<>());
```

Comment:
Suggest adding a convenience constructor that calls `this(config, context, catalogId, new ConcurrentHashMap<>())` instead of requiring the test to handle the complexity.